argH.tfa

```diff
@@ -158,3 +158,33 @@ GGAGCTGACATCGGCGGCCGTCTACGTGCCGGCCGATCCCGAAACGATCAGGTGGCCACG
 CTGTTTCGGATGTGGCTGCGCGACGCGGCCGGACGCATCTCCGAGGGCGTGCTCGACGTC
 GTGCAGGCATTGGCGGATCAAGCCGGTGCACATCCGGATGCGATCATGCCGGGCAAGACT
 CATTTGCAGGCCGCCCAACCGGTGCTGCT
+>argH_17_WT
```

Comment:
Could you remove the `_WT` part from here, since the identifiers here are in the format **[gene]_[allele number]** (and shouldn't contain any other information).